scripts/linux/pkg/deb/assets/build.sh

```diff
@@ -2,6 +2,13 @@
 
 ASSETS_DIR=/assets
 DEST_DIR=/build
+release=$(lsb_release -sr | cut -d. -f1)
+
+if [[ $release -ge 22 ]]; then
+    sed -i 's/dh-systemd (>=1.5)/debhelper (>= 9.20160709)/g' $ASSETS_DIR/ravendb/debian/control
```

Comment:
Let's make separate control files for diff Ubuntu versions and suffix those with version e.g. `control.22.04` and `control.default`. Then switch over the version, move the appropriate file to `control` and remove all other control ones.